src/main/java/core/basesyntax/Application.java

```diff
@@ -3,12 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         Lottery lottery = new Lottery();
-        Ball firstBall = lottery.getRandomBall();
-        Ball secondBall = lottery.getRandomBall();
-        Ball thirdBall = lottery.getRandomBall();
 
-        System.out.println(firstBall.toString());
-        System.out.println(secondBall.toString());
-        System.out.println(thirdBall.toString());
+        Ball[] balls = new Ball[3];
```

Comment:
better create constant for magic numbers, you can read more about constants here - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names